packages/svgs/src/scripts/load-icons.ts

```diff
@@ -0,0 +1,93 @@
+import * as path from "path";
+import * as fs from "fs";
+import camelCase from "camelcase";
+import {
+    ICONS_SIZES 
+} from "./constants.ts";
+import { getAllFiles } from "./helper.ts";
+
+interface DataItem {
+    name: string;
+    group: number;
+}
+
+export interface Icon {
+    content: string;
+    group: number;
+    filePath: string;
+    name: string;
+}
+
+const getFiles = (dir: string): Promise<string[]> => {
+    return new Promise((resolve, reject) => {
+        if (fs.existsSync(dir)) {
+            resolve(getAllFiles(dir));
+        } else {
+            reject(`no such directory ${dir}`);
+        }
+    });
+};
+
+const checkSameName = (data: DataItem[]): void => {
+    const sizes = ICONS_SIZES;
```

Comment:
`const sizes = ICONS_SIZES;` is unecessary? ICONS_SIZES.map directly?

Reply:
true